Doc/library/collections.abc.rst

```diff
@@ -213,6 +213,9 @@ Collections Abstract Base Classes -- Detailed Descriptions
 
    ABC for classes that provide the :meth:`~object.__call__` method.
 
+   See :ref:`annotating-callables` for details on how to use
```

Comment:
I don't know if it was already addressed (opening the comments one by one is a bit long) but for consistency I would say "Refer to [...]" since you're using this formulation for the other cases.

Reply:
I actually suggested the opposite in https://github.com/python/cpython/pull/123544#issuecomment-2323262134, as we use "See" in existing notes like this in `typing.rst` 😄

Reply:
Alright, two agreeing opinions are enough to convince me here, updated:)
I decided to stick with "Refer to" initially because it still fits one line and doesn't add significant overhead, but consistency is more important.